assets/xml/objects/object_zl2.xml

```diff
@@ -0,0 +1,42 @@
+<Root>
+    <File Name="object_zl2" Segment="6">
+        <!-- Zelda 2 skeleton -->
+        <Skeleton Name="gZelda2Skel" Type="Flex" LimbType="Standard" Offset="0x10D70"/>
+
+        <!-- Zelda 2 eyes textures -->
+        <Texture Name="gZelda2EyesPalette" OutName="zelda_2_eyes_palette" Format="rgb5a1" Width="16" Height="16" Offset="0x2B00"/>
```

Comment:
I don't believe we have discussed a suffix for this yet, and "palette" seems a bit long, so we should probably bring it up in discord and see what people think (something like "pal" might be good enough but it's worth a discussion either way)

Reply:
Given that we'll need to support PAL/NTSC versions of objects, that's probably the one abbreviation we don't want to use!

Reply:
its important to consider, but i think context helps. and fwiw they originally used the pal abbreviation as well (see skyboxes)

Reply:
We should get back to this discussion.
I still think `Pal` is fine and doesnt conflict with the region abbreviation at all (both context and capitalization make it clear imo), but if more people want TLUT then we should settle on what the majority wants

Reply:
guess this conversation isnt really going to progress more, so TLUT is probably what the majority of people were favoring